components/infobox/wikis/stormgate/infobox_building_custom.lua

```diff
@@ -51,16 +52,16 @@ end
 function CustomInjector:parse(id, widgets)
 	local caller = self.caller
 	local args = caller.args
+	local SEP = '&nbsp;&nbsp;/&nbsp;&nbsp;'
```

Comment:
move to const vars up top
and rename to HOTKEY_SEPERATOR

Reply:
done